### DISCUSSION TITLE
Plans to create an alternative for vue?

### DISCUSSION THREAD

**Discussion by mklueh** [resend/react-email#571] [Q&A] [answered]
Hi there,

I love the idea of this project and I am looking for a while now for such a thing for Vue. 

Are there any plans to create such a package as well?

**Comment by bukinoshita** [comment, marked as answer]
> Hi there,
> 
> I love the idea of this project and I am looking for a while now for such a thing for Vue.
> 
> Are there any plans to create such a package as well?

We would love to support vue, but we want to nailed down with react first.

**Reply by maximepvrt** [reply]
@bukinoshita I just wanted to check in and see if support for Vue is now being considered for the upcoming year?

**Comment by antonreshetov** [comment]
@mklueh Check this repository - https://github.com/mysigmail/vue-email

**Comment by Flowko** [comment]
for anyone interseted in this, we have a vue3/nuxt3 librray that is well supported and well documented:
🐙 [Github](https://github.com/Dave136/vue-email)
📖 [Docs](https://vue-email.vercel.app/)
📦 [NPM](https://www.npmjs.com/package/vue-email)
🧪 [Demo](https://vue-email-demo.vercel.app/)
